packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html

```diff
@@ -8,9 +8,254 @@
     <div class="row py-4">
         <div class="col-md-4">
             <oui-tile
-                data-heading="{{:: 'pci_notebooks_general_information_info_title' | translate }}"
+                data-heading="{{ 'pci_notebooks_general_information_info_title' | translate }}"
             >
-                <oui-tile-description></oui-tile-description>
+
+                <!--Name-->
+                <oui-tile-definition
+                    data-term="{{:: 'pci_notebooks_general_information_info_notebook_name' | translate }}"
+                    data-description="{{:: $ctrl.notebook.name }}"
+                ></oui-tile-definition>
+
+                <!--Editor-->
+                <oui-tile-definition
+                    data-term="{{:: 'pci_notebooks_general_information_info_notebook_editor_code' | translate }}"
+                    data-description="{{:: $ctrl.notebook.spec.env.editorId }}"
+                ></oui-tile-definition>
+
+                <!--Container-->
+                <oui-tile-definition
+                    data-term="{{:: 'pci_notebooks_general_information_info_notebook_container' | translate }}"
+                >
+                    <!--Nb total container-->
+                    <oui-tile-description>
+                        <span
+                            data-translate="pci_notebooks_general_information_info_notebook_container_total"
+                            data-translate-values="{ xContainer: $ctrl.notebook.volumes.length }"
+                        ></span>
+                    </oui-tile-description>
+
+                    <!--Nb swift container-->
+                    <oui-tile-description>
+                        <span
+                            data-translate="pci_notebooks_general_information_info_notebook_container_swift_total"
+                            data-translate-values="{ xContainer: $ctrl.notebook.geNbContainerByType($ctrl.NOTEBOOK_VOLUME_TYPE.SWIFT) }"
```

Comment:
There is a typo here. It should be `ge't'NbContainerByType`

Reply:
don't need it anymore

Reply:
removed